apps/checkout/src/sections/CheckoutForm/CheckoutForm.tsx

```diff
@@ -0,0 +1,69 @@
+import { Divider } from "@/components/Divider";
+import { useCheckout } from "@/hooks/useCheckout";
+import { Contact } from "@/sections/Contact";
+import { ShippingMethods } from "@/sections/ShippingMethods";
+import { UserAddresses } from "@/sections/UserAddresses";
+import { useErrorMessages } from "@/hooks/useErrorMessages";
+import { useValidationResolver } from "@/lib/utils";
+import { Suspense } from "react";
+import { FormProvider, useForm } from "react-hook-form";
+import { object, string } from "yup";
+import { Button } from "@/components/Button";
+import { useCheckoutFinalize } from "./useCheckoutFinalize";
+import { FormData } from "./types";
+import { useFormattedMessages } from "@/hooks/useFormattedMessages";
+
+export const CheckoutForm = () => {
+  const formatMessage = useFormattedMessages();
+  const { errorMessages } = useErrorMessages();
+  const { checkout } = useCheckout();
+  const { checkoutFinalize } = useCheckoutFinalize();
+
+  // TMP
+  // const [selectedPaymentProvider, setSelectedPaymentProvider] =
+  //   useState<string>();
```

Comment:
intended to stay?

Reply:
intended, this will be needed back when we display payment options